SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-or-later

tinycc's extern support is bad. i.e. these externs don't propagate through.

--- builtins/common.c	2021-01-15 21:32:56.938683418 +1100
+++ builtins/common.c	2021-01-15 21:34:09.102153806 +1100
@@ -46,6 +46,9 @@
 #include "../flags.h"
 #include "../jobs.h"
 #include "../builtins.h"
+int num_shell_builtins;
+struct builtin *shell_builtins;
+struct builtin *current_builtin;
 #include "../input.h"
 #include "../execute_cmd.h"
 #include "../trap.h"
